docs/reference/search/rrf.asciidoc

```diff
@@ -516,6 +515,157 @@ We end with `_id: 3` as `_rank: 1`, `_id: 2` as `_rank: 2`, and
+                [
+                    {
+                        "value": 2,                                                                                                                                               <4>
+                        "description": "rrf score: [0.33333334], for rank [2] in query at index [0] computed as [1 / (2 + 1]), for matching query with score: ",
+                        "details":                                                                                                                                                <5>
+                        [
+                            {
+                                "value": 0.15876243,
+                                "description": "weight(text:rrf in 0) [PerFieldSimilarity], result of:",
+                                "details":
+                                [
+                                    ...
+                                ]
+                            }
+                        ]
+                    },
+                    {
+                        "value": 1,                                                                                                                                              <6>
+                        "description": "rrf score: [0.5], for rank [1] in query at index [1] computed as [1 / (1 + 1]), for matching query with score: ",
+                        "details":
+                        [
+                            {
+                                "value": 1,
+                                "description": "within top k documents",
+                                "details":
+                                []
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+        ...
+    ]
+}
+----
+// NOTCONSOLE
+
+<1> the final RRF score for document with `_id=3`
+<2> a description on how this score was computed based on the ranks of this document in each individual query
+<3> details on how the RRF score was computed for each of the queries
+<4> the `value` heres specifies the `rank` of this document in the specific query
+<5> standard `explain` output of the underlying query, describing matching terms and weights
+<6> the `value` heres specifies the `rank` of this document for the second (`knn`) query
+
+In addition to the above, explain in RRF also supports *named queries* using the `_name` parameter.
+This however is not available for retrievers yet, so it is only supported for RRF through the use of  `sub_searches`,
+and for top-level `query` and `knn` elements for now. Using named queries allows for easier and more intuitive understanding of the RRF score computation,
+especially when dealing with multiple queries. So, for the equivalent of the above example, we would now have:
+
+[source,js]
+----
+GET example-index/_search
+{
+    "sub_searches": [
```

Comment:
sub_searches is going away right with retrievers? We should not add any new documentation or features using it. We need to focus on retrievers.